pages/papers/how-to-create-gems.mdx

```diff
@@ -1,39 +1,40 @@
+  possible. Remember, you're selling both the organizers of the events to select
+  your talk, as well as trying to convince attendees your talk is the one they
+  should see.
 </div>
 
-Si quieres empezar a contribuir a una comunidad creo que el primer gran paso seria creando tu propia gema, independientemente si soluciona un problema especifico para un monton de personas, o si solo lo haces para pasarla bien y divertirte haciendolo, o simplemente por curiosidad te inicias con un genial "Hola mundo, vengo en son de paz", son muchos los motivos por los cuales uno puede animarse a construir una gema, lo ideal seria identificar eso que siempre repites en todos los proyectos que pasaron por tus manos, y automatizarlo, como? construyendo una gema e invocandolo en todos los proyectos que necesites.
+If you want to start contributing to a community I think the first big step would be creating your own gem, regardless of whether it solves a specific problem for a lot of people, or if you just do it to have fun and have fun doing it, or just out of curiosity you start with a great "Hello world, I come in peace", there are many reasons why one can be encouraged to build a gem, the ideal would be to identify what you always repeat in all the projects that passed through your hands, and automate it, building a gem and invoking it in all the projects you need.
 
-Entonces, ruby es un lenguaje potente, y muy claro de leer, construir una gema es lo mas divertido del mundo. Aprenderemos a como crear gemas, con ideas locas y divertidas, quizas no le sirva a nadie, pero estoy seguro que nos divertiremos en el proceso.
+So, ruby is a powerful language, and very clear to read, building a gem is the most fun thing in the world. We will learn how to create gems, with crazy and funny ideas, maybe it will not help anyone, but I am sure we will have fun in the process.
 
 ### Notes
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
- This is where you should explain things such as technical requirements, why you're the best person to speak on this subject, etc...
+  This is where you should explain things such as technical requirements, why
+  you're the best person to speak on this subject, etc...
 </div>
 
-Soy estudiante en Codeable un bootcamp de fullstack web developer donde construimos cosas increibles usando ruby, react y mas. Estoy ilusionado por contar cómo me ha cambiado la vida compartir recursos con la comunidad, mi intencion es contagiar toda esa emocion para que cada vez mas personas puedan empezar su camino en la programacion con tecnologias ruby y todo ese ecosistema.
+I am a student at Codeable, a bootcamp of fullstack web developer where we build amazing things using ruby, react and more. I am excited to tell how my life has changed to share resources with the community, my intention is to spread all that emotion so that more and more people can start their journey in programming with ruby technologies and all that ecosystem.
```

Comment:
developers* [...] Ruby, React, and more. I am excited to tell how sharing resources with the community has changed my life [...] Ruby* [...]

Reply:
Optional: where we build applications similar to Netflix, Slack, Discord and Trivago.